src/pages/Blog/BlogItem/BlogItem.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC } from "react";
+import { useLoaderData } from "react-router-dom";
+
+const BlogItem: FC = () => {
+  const data = useLoaderData();
+  if (!data) {
+    return <div>Loading...</div>;
+  }
+  const hasAuthor = (value: unknown): value is { author: string } =>
+    typeof value === "object" && value !== null && "author" in value;
+  if (!hasAuthor(data)) {
+    throw new Error("Invalid data");
+  }
+
+  const { author, ...post } = data;
+  if (!isPost(post)) {
+    throw new Error("Invalid post");
```

Comment:
I guess we will need to implement some kind of error handling, because this one here will break the whole app

Reply:
![good-evil](https://user-images.githubusercontent.com/51704650/205335167-d71efcc0-488d-4814-bb82-15118172ee6d.gif)